.github/workflows/test-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Development Release - Linux
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+on:
+  workflow_dispatch:
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.BRANCH_NAME }}
+          fetch-depth: '0'
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Gradle Build
+        run: |
+          ./gradlew build --info --stacktrace
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
+      - name: Publish package
+        uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
+        with:
+          arguments: publish
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Looks like it actually publishes the package. From the name of the workflow I thought this would be `test-release.yml` meaning "test if the release would work"...

Reply:
Yeah. This is definitely a test release. :) But it publishes to Github Packages. Not the Maven Central. If this works out, I will add Maven Central to it.